### PR TITLE
Added API_KEY and device id to all examples.

### PR DESCRIPTION
**Pull request by Vinayakj009**
Have added API_KEY and DEVICE_ID macro parameter to all examples, so that users can quickly change this parameter, and accordingly use it with their codes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,25 @@ Please follow the steps to include the bolt library to arduino
...
 2. Open Arduino IDE.
 3. Click on Sketch->Add Library->Add .Zip library.
-4. Navigate to the folder where boltiot-arduino-helper.zip was downloaded, and select boltiot-arduino-helper.zip.
+4. Navigate to the folder where boltiot-arduino-helper-master.zip was downloaded, and select boltiot-arduino-helper-master.zip.
 5. Press ok and the library is loaded to your arduino ide.
 
+# (Optional) Defining global API key and device id (Details of why this is needed will be added later)
+While it is a good idea to define the API key and device id as a part of the code, it is sometimes troublesome to write the corresponding code everytime you have a new project.
+You can define a global API key or device id, by using the following steps.
+1. Go to the 'libraries/boltiot-arduino-helper-master/src' folder inside your sketchbook folder. The sketchbook folder is normally 'My Documents/Arduino' folder for windows and "~/Documents/Arudino/" folder in Mac and linux.
```
Comment (sashoeb94) [inline README.md]: @Vinayakj009 Typo in windows installation path "~/Documents/Arudino"

**Comment by gopherengine** [comment]
I hav'nt got the Ok button after importing the lib.

File: examples/CommandHandler/CommandHandler.ino [hunk 2]
```diff
@@ -7,6 +7,12 @@
  * The User will also be able to parse the response of the command 
  * using the SerialRead API
  */
+#ifndef API_KEY
+#define API_KEY   "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+#endif
+#ifndef DEVICE_ID
+#define DEVICE_ID "BOLTxxxxxxxx"
```
Comment (rsbuilds) [inline examples/CommandHandler/CommandHandler.ino:14]: @Vinayakj009  Can we make #define DEVICE_ID some sort of list? Where I can store the multiple devices.

File: library.properties [hunk 3]
```diff
@@ -1,10 +1,10 @@
 name=BoltIoT-Arduino-Helper
-version=1.1.2
+version=1.1.3
 author=Inventrom
 maintainer=Inventrom
 sentence=Enables communication and control over the Bolt WiFi module
 paragraph= This library has been developed to reduce the effort required to interface the Arduino with the Bolt IoT platform.
 category=Communication
-url=https://cloud.boltiot.com/documentation
+url=https://cloud.boltiot.com/docs
```
Comment (rsbuilds) [inline library.properties:8]: @Vinayakj009 Confirm with @pranavkundaikar if it should be docs.boltiot.com

File: examples/CommandHandler/CommandHandler.ino (hunk 2 above)
Comment (Vinayakj009) [reply examples/CommandHandler/CommandHandler.ino:14]: While a list can be defined, having the OTA update recognize the list, is not a part of the repo.
Also, I am planning on the ability to define a product id, such that all devices linked to the product will get OTA update.

File: library.properties (hunk 3 above)
Comment (Vinayakj009) [reply library.properties:8]: I do not think I should change it here.
This link already re-directs to the new documentation.

Also, I feel we should make changes in the Bolt Cloud such that, all old links re-direct to the new documentation, rather then making changes in where we have placed the links.

**Review by rsbuilds — APPROVED** [review]

**Review by sashoeb94 — APPROVED** [review]

**Pull request merged**